pkg/sessionctx/stmtctx/stmtctx.go

```diff
@@ -1517,3 +1559,21 @@ func GetStmtLabel(ctx context.Context, node ast.StmtNode) string {
 	}
 	return ast.GetStmtLabel(node)
 }
+
+// TableDelta stands for the changed count for one table or partition.
+type TableDelta struct {
+	Delta    int64
+	Count    int64
+	InitTime time.Time // InitTime is the time that this delta is generated.
+	TableID  int64
```

Comment:
nit: maybe remove it from the old code? Seems callers all use it as map key. We can do it in another PR.